packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -732,6 +740,7 @@ export class RunningSummarizer implements IDisposable {
 			this.enqueuedSummary.resultsBuilder.fail(
 				"Aborted; overridden by another enqueue summarize attempt",
 				undefined,
+				"base",
```

Comment:
Should we get another Stage for this kind of thing, where the machinery just couldn't even get started?

Reply:
Yeah, "none" maybe.

Reply:
Removed all of the changes in this file.